tools/cldr-code/src/main/java/org/unicode/cldr/util/VoterReportStatus.java

```diff
@@ -38,8 +40,13 @@ static ReportAcceptability fromPair(boolean isComplete, boolean isAcceptable) {
     public static class ReportStatus {
         public EnumSet<ReportId> completed = EnumSet.noneOf(ReportId.class);
         public EnumSet<ReportId> acceptable = EnumSet.noneOf(ReportId.class);
+        public Date date = null;
```

Comment:
could be private?

Reply:
this class is used more like a struct, but it could be private here. With getters for the other fields as well.